lib/apiservers/engine/backends/eventmonitor.go

```diff
@@ -257,17 +255,17 @@ func (p DockerEventPublisher) PublishEvent(event plevents.BaseEvent) {
 
 				err := NewContainerBackend().ContainerRm(vc.Name, config)
 				if err != nil {
-					log.Warnf("failed to auto remove container: %s", err.Error())
+					log.Errorf("Failed to remove container %s: %s", vc.ContainerID, err)
 				}
 			}
-
 		}()
 	case plevents.ContainerRemoved:
 		attrs = make(map[string]string)
-		//pop the destroy event...
+		// pop the destroy event...
 		actor := CreateContainerEventActorWithAttributes(vc, attrs)
 		EventService().Log(containerDestroyEvent, eventtypes.ContainerEventType, actor)
 		if err := UnmapPorts(vc.ContainerID, vc.HostConfig); err != nil {
+			log.Errorf("Failed to unmap ports for container %s: %s", vc.ContainerID, err)
 			log.Warn(err)
```

Comment:
Remove this since `err` is logged twice.

Reply:
Done